bin/node/src/chain_spec.rs

```diff
@@ -361,6 +361,8 @@ fn generate_genesis_config(
     let accounts_config =
         configure_chain_spec_fields(unique_accounts_balances, authorities, controller_accounts);
 
+    let reserved = accounts_config.members[0..2].to_vec();
```

Comment:
okay, I see below that it was already this way, but still, I think it won't hurt

Reply:
The thing is for local runs, we need some positive value in order to get that feature working. Other solution would be to set it to 0, but then anyone to get committee rotated would need to issue `change_reserved_members`. When I'm thinking it now I think this is better @kostekIV . This is because we want to be as close as possible to Testnet and Mainnet deployments, and those deployments would require such manual steps anyway.

Reply:
That makes sense, but would also make e2e tests a little bit harder, with a few more setup steps:
* issue change_reserved_members
* wait an era
* actual test

Reply:
I think it's the price we can pay for avoiding mistakes during releases.

Reply:
@pmikolajczyk41  feel free to resolve, from my end it looks okay